build.gradle

```diff
@@ -1,13 +1,26 @@
 buildscript {
-    ext.kotlin_version = '1.3.11'
+    ext {
+        supportLibraryVersion = '27.1.1'
```

Comment:
androidx cough cough

Reply:
BTW, it's just to suggest you a new issue will come 😃

Reply:
Ough! This version constant is not used anymore, we already replaced everything with Android X, deleting it.

Reply:
Ough, we have already replaced everything with Android X so this version constant is not used anymore, deleting it...